.github/workflows/bot.yml

```diff
@@ -73,6 +73,14 @@ jobs:
         run: |
           HUDI_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
           ./packaging/bundle-validation/ci_run.sh $HUDI_VERSION
+      - name: Common Test
```

Comment:
@Zouxxyy what are we specifically looking for to be tested in here? We need to be careful in expanding the scope here

Reply:
From the cmd, it seems to test the `hudi-common` module specifically.

Reply:
@alexeykudinkin Because the patch repairs the avro compatibility of the test cases in the hudi-common module under spark3, the test of the hudi-common module is added in bot

Reply:
@Zouxxyy my point being -- hudi-common components (or tests) should not depend on particular Spark version. If something like in this case relies on particular Spark version (or other dependency that Spark brings) we should move it from hudi-common to hudi-spark

Reply:
there is a deeper issue with this - hudi common is tightly coupled with avro models, which variates wrt spark profiles. currently hudi-common jar won't be compatible across all engine profiles: e.g., if built with spark3.3 (avro 1.11), it won't work with spark 2 (avro 1.8)  or flink (avro 1.10). this needs to be decoupled first.